alpha/declcfg/load.go

```diff
@@ -168,3 +168,18 @@ func readYAMLOrJSON(r io.Reader) (*DeclarativeConfig, error) {
 	}
 	return cfg, nil
 }
+
+// LoadFile will unmarshall declarative config components from a single 'file' located
+// at a filesystem hierarchy 'root' represented in string form as 'path'
+func LoadFile(root fs.FS, file io.Reader, path string) (*DeclarativeConfig, error) {
+	cfg, err := readYAMLOrJSON(file)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := readBundleObjects(cfg.Bundles, root, path); err != nil {
+		return nil, err
```

Comment:
To keep the same behavior as before, should we keep the same error context?
```suggestion
		return nil, fmt.Errorf("read bundle objects: %v", err)
```